.github/workflows/main.yml

```diff
@@ -1,6 +1,11 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  repository_dispatch:
```

Comment:
This is for manually triggering a build, which can be done with a VSCode extension for instance: https://github.com/cschleiden/vscode-github-actions#trigger-runs